torchtitan/parallelisms/pipelining_utils.py

```diff
@@ -19,7 +18,7 @@ def build_pipeline_schedule(job_config, stages, loss_fn):
     schedule_class = get_schedule_class(
         job_config.experimental.pipeline_parallel_schedule
     )
-    if schedule_class in [ScheduleInterleaved1F1B, ScheduleFlexibleInterleaved1F1B]:
+    if issubclass(schedule_class, PipelineScheduleMulti):
         looped_schedule = True
```

Comment:
```suggestion
    looped_schedule = issubclass(schedule_class, PipelineScheduleMulti)
```
and then we can remove line 16 init of this variable.